src/react/ui-elements/Veeam/VeeamSkipModal.tsx

```diff
@@ -0,0 +1,54 @@
+import { Modal, Stack, Wrap } from '@scality/core-ui';
+import { Button } from '@scality/core-ui/dist/next';
+import styled from 'styled-components';
+
+type VeeamSkipModalProps = {
+  isOpen: boolean;
+  close: () => void;
+  exitAction: () => void;
+  modalContent: JSX.Element;
+  title?: string;
+};
+
+const ModalContent = styled.div`
+  max-width: 30rem;
+`;
+
+export const VeeamSkipModal = ({
```

Comment:
nit: Maybe we should add a test to make sure this confirm modal is opened when clicking on `Skip Use case configuration `

Reply:
Done in f00d1b3fc0b1d110e9937196d6dd4f490010f362